sdk/cosmosdb/cosmos/src/queryExecutionContext/Aggregators/CountAggregator.ts

```diff
@@ -1,7 +1,7 @@
 import { IAggregator } from "./IAggregator";
 
 /** @hidden */
-export class CountAggregator implements IAggregator<number> {
+export class CountAggregator implements IAggregator {
   public value: number;
```

Comment:
The aggregate could be not a number right?

Reply:
Yes. I removed it since item2 is now sometimes an object